README.md

```diff
@@ -80,8 +80,7 @@ Custom attr:
  * `pstsPaddingMiddle` If true, the tabs start at the middle of the view (Like Newsstand google app)
  * `pstsTextStyle` Set the text style, default bold
  * `pstsTextSelectedStyle` Set the text style of the selected tab, default bold
- * `pstsTextAlpha` Set the text alpha transparency, default 0.5
- * `pstsTextSelectedAlpha` Set the text alpha transparency of the selected tab, default 1
+ * `pstsTextColorInactive` Set text color of inactive text so you can use ARGB without using `View.setAlpha`
```

Comment:
It makes sense, but then you always have to define a transparency colour in `android:textColor` which is kind of weird.... What about creating `pstsTextColorActive ` and `pstsTextColorInactive `? There is no need to rely on `android:textColor` , It will clarify things in my opinion. Because with that many attr, devs used to look for custom attr to costumize the tabs. Does it make sense to you?

If nothing is defined we use `textPrimaryColor`. Should we ignore `android:textColor` if it is defined?